integrationTests/testProcessorNode.go

```diff
@@ -1364,6 +1367,10 @@ func (tpn *TestProcessorNode) initMetaInnerProcessors() {
 	}
 	tpn.TxProcessor, _ = transaction.NewMetaTxProcessor(argsNewMetaTxProc)
 
+	//TODO for the next task: call this automatically from somewhere else (systemSCs.go) for each and every new system SC that we will define
```

Comment:
does not make sense to call that one at genesis. not all the nodes have it.

Reply:
The original productive code did call this at genesis. 
Will make a new function for this and call it in the required tests